src/Tokenizer/Token.php

```diff
@@ -495,6 +517,7 @@ public function setContent($content)
 
         // setting empty content is clearing the token
         if ('' === $content) {
+            @trigger_error(__METHOD__.' shall not be use to clear token, use Tokens::clearAt instead.', E_USER_DEPRECATED);
```

Comment:
Then isn't this deprecation notice useless as we already have the one for the method deprecation itself?